metrics-exporter-prometheus/src/lib.rs

```diff
@@ -101,9 +101,10 @@
 #![deny(missing_docs)]
 #![cfg_attr(docsrs, feature(doc_cfg), deny(rustdoc::broken_intra_doc_links))]
 mod common;
-pub use self::common::{BuildError, Matcher};
+pub use self::common::{BuildError, Matcher, sanitize_description, sanitize_label_key, sanitize_label_value, sanitize_metric_name};
```

Comment:
I think what I'd want to do here is move all of these `sanitize_*` free functions into a new module called `rendering` at the top level, and then expose the module publicly.

That gives us a place to expose more of the rendering code in the future without cluttering the top level of the crate, in terms of what the docs end up looking like.

Reply:
Good idea. I can do that.

Reply:
Maybe we can call it formatting @tobz ?

Reply:
Yeah, I'm fine with `formatting`.

Reply:
I also moved the write_line functions to this module. They all have the same thing in common, formatting strings and sanitizing them for Prometheus.